pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--ignore=panda/ --ignore=rednose_repo/ --ignore=tinygrad_repo/ --ignore=laika_repo/"
+python_files = "test_*.py"
+timeout = "30"  # you get this long by default
```

Comment:
seeing this warning: `/home/batman/.pyenv/versions/3.11.4/lib/python3.11/site-packages/_pytest/config/__init__.py:1373: PytestConfigWarning: Unknown config option: timeout`

Reply:
waiting for the poetry work to get in before adding the package, you can ignore for now